packages/gatsby-theme-docs/utils/api-tag-info.js

```diff
@@ -0,0 +1,25 @@
+const unified = require('unified');
```

Comment:
@nkuehn didn't get that far... unfortunately.

Limiting to only methods.mdx file for the moment. If I attempt to parse the file as is. It fails with an error in one of the remark-mdx dependencies.

If I simplify the code removing the `\n` in `ApiEndpoint` tag. I only get the headings tags in the AST.

any ideas?
You can try it out by pulling the branch and running `rm -rf .cache && yarn start` in the `api-docs-smoke-test` microsite

Reply:
thank you @nkuehn.
re the missing topic folder, I have solved adding an empty topic folder to the api microsite locally (forgotten about git and empty folders 🤦 ).

I had had a look to the `createProcessor` but in the example https://github.com/unifiedjs/unified#processorparsefile seemed to do exactly the same thing I was doing using unified, so I didn't try it.
As you say, it's probably a side effect of non compatible versions.

I'll move on tomorrow starting from here. Thank you.

Reply:
If we require folders we generally autogenerate them at startup, too to avoid confusing errors for authors.   At the top of `gatsby-node.js` there's some code that ensures the minimum conventional filesystem structure for a site. 

The docs about the processor API you linked are actually weird and a bit misleading because if you get a processor instance you don't actually have to do the chaining starting from `unified().`.  Maybe they just copy / pasted some documentation :-)